docs/agents.md

```diff
@@ -20,36 +20,70 @@ In typing terms, agents are generic in their dependency and output types, e.g.,
 
 Here's a toy example of an agent that simulates a roulette wheel:
 
-```python {title="roulette_wheel.py"}
-from pydantic_ai import Agent, RunContext
+=== "Pydantic AI Gateway"
```

Comment:
I think this should be the second tab, so we don't make things unnecessarily complicated for new users

Reply:
If we keep it like this, we should at least link to the gateway docs

Reply:
Added and info box with

> "Code Examples with Gateway Support"